### PR TITLE
Add FAQ page

### LINKED ISSUE #843: Add a FAQ page
**Is your feature request related to a problem? Please describe.**
We receive a high number of emails. Despite the chat bot, it would be interesting to have an FAQ page that displays frequently asked questions.

**Describe the solution you'd like**
Add a FAQ page that would look something like this : 
image

The page could also be in the "More" tab.

### PR DESCRIPTION
**Pull request by camillebrulotte**
### ⁉️ Related Issue
Closes #843 

## 📖 Description
The FAQ page displays frequently asked questions.

### 🧪 How Has This Been Tested?
By clicking on the 'Need help' button in the home page not logged in and clicking on 'Need help' in the more page.

### ☑️ Checklist before requesting a review
- [x] I have performed a self-review of my code.
- [x] If it is a core feature, I have added thorough tests.
- [ ] If needed, I added analytics.
- [x] Make sure to add either one of the following labels: `version: Major`,`version: Minor` or `version: Patch`. 
- [x] Make sure golden files changes were reviewed and approved.

### 🖼️ Screenshots (if useful):
New item in the list : 
image
image
image

New page in the login : 
image
image
image

### CODE REVIEW

File: test/ui/views/faq_view_test.dart [hunk 1]
```diff
@@ -1,14 +1,13 @@
...
 import 'package:flutter/material.dart';
 import 'package:flutter_test/flutter_test.dart';
 import 'package:flutter_gen/gen_l10n/app_localizations.dart';
 
 // SERVICES
-import '../../mock/managers/settings_manager_mock.dart';
 
 // MANAGERS
 import 'package:notredame/core/managers/settings_manager.dart';
+import '../../mock/managers/settings_manager_mock.dart';
```
Comment (JonathanDuvalV) [inline test/ui/views/faq_view_test.dart:8]: Je penses que les mocks ont leur propre section habituellement genre // MOCKS 😮

File: test/viewmodels/faq_viewmodel_test.dart [hunk 2]
```diff
@@ -0,0 +1,53 @@
+// FLUTTER / DART / THIRD-PARTIES
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:mockito/mockito.dart';
+
+// SERVICES / MANAGERS
+import 'package:notredame/core/services/launch_url_service.dart';
+import 'package:notredame/core/managers/settings_manager.dart';
+import '../mock/services/launch_url_service_mock.dart';
```
Comment (JonathanDuvalV) [inline test/viewmodels/faq_viewmodel_test.dart:9]: Celui la aussi 🚨

**Review by JonathanDuvalV — APPROVED** [review]
LGTM

**Review by Udito3 — APPROVED** [review]
LGTM!

**Pull request merged**